blog/closed-beta.md

```
@@ -29,6 +44,65 @@ Lenders can customize default handling, choosing between collateral repossession
+1. View markets and explore the Creditbook
+2. Discover new collateral options
+3. Start a loan by borrowing funds
+4. Check out your portfolio
+   ‍
+   For a more detailed guide on using the platform, please refer to our product documentation.
+   ‍
+### How else can I get involved?
+
+We encourage you to join our Discord community and follow us on Twitter to stay up-to-date with the latest news and developments. Message us for a Closed Beta access code, and waitlisted users can still refer other users to boost their position on the waitlist and earn points.k tokens, whiskey traders leveraging existing inventory for purchase orders, and early employees at protocols seeking liquidity on locked assets.
+‍
+## Introducing Loopscale
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)